electron-app/src/components/settings/api-keys/ExternalServices.vue

```diff
@@ -1,8 +1,18 @@
 <template>
-  <div>
+  <v-container>
     <v-row>
       <v-col cols="12">
-        <h2>External Services</h2>
+        <v-card>
+          <v-card-title>External Services</v-card-title>
+          <v-card-text>
+            <p>
+              Rotki can connect to service providers in order to obtain more
```

Comment:
I would rephrase to:

Rotki connects to various service providers in order to obtain information such as historical prices or blockchain data. In certain cases Rotki depends on these APIs for basic information, in which case you will need to provide an API key.